docs/getting_started/setup/windows.md

```diff
@@ -0,0 +1,22 @@
+Here's how you can install Crow on your Windows machine.
+# Getting and Compiling Crow
+## Using A package manager
+### VCPKG
```

Comment:
I think a section for CMake should be added. (maybe this is a later PR though)

Just like on Linux, `find_package(Crow CONFIG REQUIRED)` should work on windows with vcpkg.

Reply:
Yes, same for MacOS, I'll try to add instructions in another PR